plugins/git-release-manager/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@backstage/plugin-git-release-manager",
-  "version": "0.1.3",
+  "version": "0.2.0",
```

Comment:
Hmm. This version bump is something that changeset bot handles, along with generating the changelog and publishing the package to npm. It's done when this Version Packages PR is merged. https://github.com/backstage/backstage/pull/6507

So, I don't think you need to change the version, as long as you have a changeset in this PR. :)

Reply:
I had a suspicious, but wasn't sure 😅

I'll revert this bump :)

Reply:
Noice!